packages/hash/shared/src/ProsemirrorManager.ts

```diff
@@ -429,13 +430,16 @@ export class ProsemirrorManager {
 
     // If the target entity is the same as the block's child entity
     // we don't need to do anything
-    if (targetChildEntity.entityId === childEntity.entityId) {
+    if (
```

Comment:
Do we want to compare editionId's here? or baseId's?

Reply:
I think we can compare on editionIds here, will add the check for version as well (we can't rely on object equivalence without a deepcompare)